<?xml version="1.0" encoding="utf-8"?>

<launch>
  <arg name="trackker_runner_settings"              default="LGSVL"/>

  <node pkg="unscented_kf_v4" name="ukf_ros_node" type="ukf_ros_node_lgsvl.py" output="screen">
    <param name="trackker_runner_settings"        value="$(arg trackker_runner_settings)" />
  </node>
  <node pkg="lgsvl_ros_msg_converter" name="lgsvl_ros_msg_converter" type="lgsvl_gt_to_object_list.py" output="screen">
  </node>


  <!-- <node pkg="rosbag" type="play" name="player" output="log" args=" -r 1 /home/numan/Desktop/ukf_versions/simulation_ws/2021-11-19-12-57-31.bag"/> -->
  <node pkg="rosbag" type="play" name="player" output="log" args=" -r 1 /home/numan/Desktop/ukf_versions/simulation_ws/2021-09-29-13-46-02-scenario_1.bag"/>

</launch>
